Parsers/ASimNetworkSession/ARM/FullDeploymentNetworkSessions.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
We have a challenge that it would not work (well) without empty. In any case I will add additional parsers there in the near future, so indeed. why not.